docs/moderator-handbook.md

```diff
@@ -177,19 +177,19 @@ Forum moderators have the ability to delete user's posts. You should only do thi
 
 For the first spam post of a user, send them a message explaining the problem, and remove the link or post as appropriate. Leave a note on the user's profile explaining the action you have taken. If the problem persists, then quietly block the user from posting (using the silence option on the User Admin panel). Send the user a warning with the Code of Conduct. Check the box in the private message indicating that your message is a "formal warning."
 
-You can ask questions and report incidents in the in the [staff forum section](https://forum.freecodecamp.com/c/staff).
+You can ask questions and report incidents in the [staff forum section](https://forum.freecodecamp.com/c/staff).
 
 ### Dealing with off-topic conversations
 
-Posts or topics that seems to be in the wrong place can be re-categorized or renamed to whatever would be appropriate.
+Posts or topics that seem to be in the wrong place can be re-categorized or renamed to whatever would be appropriate.
 
-In exceptional circumstances, it may be appropriate for a moderator to fork a discussion into multiple threads.
+It may be appropriate for a moderator to fork a discussion into multiple threads in exceptional circumstances.
```

Comment:
I suggest reverting this change.
```suggestion
In exceptional circumstances, it may be appropriate for a moderator to fork a discussion into multiple threads.
```